src/Navigation/Dropdown/Dropdown.test.tsx

```diff
@@ -292,6 +314,61 @@ describe('<Dropdown/> keydown event', () => {
     ).toBeVisible();
   });
 
+  test('when the first option is highlighted, pressing up arrow key should not affect anything', async () => {
+    render(
+      <Dropdown label="Career">
+        <DropdownItem value="pm">Product Manager</DropdownItem>
+        <DropdownItem value="se">Software Engineer</DropdownItem>
+      </Dropdown>
+    );
+
+    const dropdown = queryByRole(document.body, 'menuitem');
+    fireEvent.click(dropdown);
+
+    const optionPM = queryByText(document.body, 'Product Manager');
+    const optionSE = queryByText(document.body, 'Software Engineer');
+    expect(optionPM).toHaveStyle(`background: ${Greyscale.softgrey}`);
```

Comment:
I guess there's no way around it without changing the implementation, but I don't think it's good that we're testing styles instead of testing logic here. If we change the styles of active elements, these tests will break.

Reply:
Checked class name instead. 🙏